recipes/kedro/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "kedro" %}
+{% set version = "0.15.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/quantumblacklabs/kedro/archive/{{ version }}.tar.gz
+  sha256: b9a1aef1f6a5aba91e8726bcc85d746f8966fcb12dd466b506890e76326d3357
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5,<3.8
+  run:
+    - anyconfig <1.0,>=0.9.7
+    - azure-storage-blob <2.0,>=1.1.0
+    - azure-storage-file <2.0,>=1.1.0
+    - azure-storage-queue <2.0,>=1.1.0
+    - click <8.0
+    - cookiecutter <2.0,>=1.6.0
+    - fsspec <0.7.0,>=0.5.1
+    - pandas <1.0,>=0.24.0
+    - pandas-gbq <1.0,>=0.12.0
+    - pip-tools <5.0.0,>=4.0.0
+    - pyarrow <1.0.0,>=0.12.0
+    - python >=3.5,<3.8
+    - python-json-logger <1.0,>=0.1.9
+    - pyyaml <6.0,>=4.2
+    - requests <3.0,>=2.20.0
+    - s3fs <0.4.1,>=0.3.0
+    - sqlalchemy <2.0,>=1.2.0
+    - pytables <3.6,>=3.4.4
+    - toposort <2.0,>=1.5
+    - xlrd <2.0,>=1.0.0
+    - xlsxwriter <2.0,>=1.0.0
+
+test:
+  imports:
+    - kedro
+
+about:
+  home: https://github.com/quantumblacklabs/kedro
+  summary: A Python library that implements software engineering best-practice for data and ML pipelines.
+  license: Apache-2.0
+  license_file: LICENSE.md
+  doc_url: https://kedro.readthedocs.io/
+  dev_url: https://github.com/quantumblacklabs/kedro
+  description: |
+    Kedro is a development workflow framework that implements
+    software engineering best-practice for data pipelines
+    with an eye towards productionising machine learning models.
+
+extra:
+  recipe-maintainers:
+    - marcelotrevisani
+    - lorenabalan
+    - yetudada
+    - idanov
```

Comment:
@lorenabalan , @yetudada , @idanov
Could you please confirm that you would like to maintain this recipe as wel?

Reply:
I can confirm! Thank you @marcelotrevisani!

Reply:
I confirm! Thanks for all your help @marcelotrevisani !

Reply:
Just waiting for the confirmation of @idanov

Reply:
I confirm. Thanks a lot @marcelotrevisani!